static/app/views/settings/project/projectProcessingIssues.tsx

```diff
@@ -320,6 +323,14 @@ class ProjectProcessingIssues extends Component<Props, State> {
         {dsymUUID && <span> {dsymUUID}</span>}
         {dsymArch && <span> {dsymArch}</span>}
         {dsymName && <span> (for {dsymName})</span>}
+        {(release || dist) && (
+          <div>
+            <Tag tooltipText={t('Latest Release')}>
```

Comment:
I don't know the content length for tooltips but this message sounds a bit too ambiguous. It can be that the release shown is not the latest.

I would rephrase it slightly:
```suggestion
            <Tag tooltipText={t('Latest Release Observed with Issue')}>
```

What do you think?

Reply:
A bit wordy, but I'll take it 👍